hw3/scripts/Homework0825.txt

```diff
@@ -0,0 +1,135 @@
+#PART 1: Test the hypothesis that the average activation difference across subjects for Words > fixation (the Words contrast at the single subject level) is greater than zero.
+
+#Mask out non-brain regions and coverage differences
+across subjects
+3dmask_tool -input export/masks/full_mask.sub-* -frac 1.0 -prefix mask
+
+#Choose the correct sub-brick
+#use 3dinfo to explore the options (pick a random participant)
+3dinfo -label export/betas/stats.sub-08_REML+tlrc.BRIK
+
+#Look up index of Words#0_Coef - this didn't work :-()
+#3dinfo -label2index ’Words#0_Coef’ export/betas/stats.sub-08_REML+tlrc.BRIK
```

Comment:
Make sure you use ordinary single quotes `'`. It looks like you have fancy quotes `’` inserted by a rich text editor. Ideally use a plain text editor, like Sublime to avoid this problem.

You may need to use the `.HEAD` file for commands like this. `.HEAD` files contain the meta-data, like labels; `.BRIK` files only contain image data.